src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -106,7 +106,7 @@ public virtual async Task InvokeAsync(HttpContext context, RequestDelegate next)
                                 throw;
                             return;
                         }
-                        if (deserializationResult?.Length == 1)
+                        if (deserializationResult is GraphQLRequest[] array && array.Length == 1)
```

Comment:
As I inderstand (looking through code of serializer) `&& array.Length == 1` condition here is unnecessary now but I'm fine to keep it to make things more explicit.

Reply:
Correct.